apps/ios/Shared/Model/NtfManager.swift

```diff
@@ -46,8 +54,8 @@ class NtfManager: NSObject, UNUserNotificationCenterDelegate, ObservableObject {
                 chatModel.ntfContactRequest = chatId
             }
         } else if let (chatId, ntfAction) = ntfCallAction(content, action) {
-            if let invitation = chatModel.callInvitations.removeValue(forKey: chatId) {
-                CallController.shared.callAction(invitation: invitation, action: ntfAction)
+            if let userId = userId, let invitation = chatModel.callInvitations.removeValue(forKey: chatId) {
+                CallController.shared.callAction(invitation: (userId, invitation), action: ntfAction)
```

Comment:
I am not yet sure why userId is needed here, but if it is it's a hack to pass it as a part of invitation parameter without making it a part of invitation. So either it should be inside invitation, or a separate parameter...

Reply:
I'm not entirelly sure yet how to make this thing good, will do in a separate PR